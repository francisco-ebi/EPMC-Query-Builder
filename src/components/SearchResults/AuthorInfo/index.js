export default function AuthorInfo({ author }) {
  return  (
    <p className="author">{author}</p>
  );
}