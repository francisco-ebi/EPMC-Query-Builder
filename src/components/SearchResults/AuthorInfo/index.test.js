import renderer from 'react-test-renderer';
import AuthorInfo from './index';

it('it creates', () => {
  const component = renderer.create(
    <AuthorInfo author="Joe"/>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});