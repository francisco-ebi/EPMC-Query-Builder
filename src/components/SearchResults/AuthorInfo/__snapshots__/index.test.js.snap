// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it creates 1`] = `
<p
  className="author"
>
  Joe
</p>
`;
